src/controllers/expenses.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+const {
+  getAllExpenses,
+  getExpenseById,
+  updateExpenses,
+  removeExpenses,
+  createNewExpense,
+} = require('../services/expenses');
+
+const { getUserById } = require('../services/users');
+
+const addNew = async(req, res) => {
+  const { userId, title, amount, category, note } = req.body;
```

Comment:
You need here only `userId`.  You could make your code sporter 👇 
```suggestion
  const { userId } = req.body;
```

And in line 25 use `req.body` with the spread operator.